src/main/java/org/zalando/nakadi/domain/Subscription.java

```diff
@@ -23,8 +17,18 @@ public Subscription(final String id, final DateTime createdAt, final Subscriptio
         super(subscriptionBase);
         this.id = id;
         this.createdAt = createdAt;
+        this.updatedAt=createdAt;
```

Comment:
I understand that it will do the trick, but initializing it here like that will introduce a hidden behavior that will cause problems at some point. I think its initialization should be more explicit.

Reply:
@Kunal-Jha I checked usage of this constructor. It's called from `SubscriptionDbRepository.createSubscription(...)` + 3 usage in tests. For me it would make sense to add one more parameter to this constructor and pass the value from the code that calls it. That would be "more natural" and will not introduce hidden behavior.